source/_integrations/folder_watcher.markdown

```diff
@@ -28,6 +30,11 @@ Configured folders must be added to [allowlist_external_dirs](/integrations/home
 Pattern matching using [fnmatch](https://docs.python.org/3.6/library/fnmatch.html) can be used to limit filesystem monitoring to only files which match the configured patterns.
 As example to monitor specific file, as example YAML and text-files add `*.yaml` and `*.txt`.
 
+## Entities
+
+This integration will besides the events also create `event` entities for the respective events that can happen.
+These entities can be used for automations instead of using the respective event.
```

Comment:
I think we should make this the main focus for the documentation (and not the old style events, which we can just fully remove from the docs IMHO).

Reply:
Modified accordingly. Hope that fits so it's understandable.